testing/ef-tests/src/macros/sanity_blocks.rs

```diff
@@ -61,10 +61,12 @@ macro_rules! test_sanity_blocks {
                         let signed_block: SignedBeaconBlock = utils::read_ssz_snappy(&block_path)
                             .expect(&format!("cannot find test asset (blocks_{i}.ssz_snappy)"));
 
-                        result = state
-                            .state_transition(&signed_block, validate_result, &mock_engine)
-                            .await
-                            .map_err(|err| err.to_string());
+                        if result.is_ok() {
+                            result = state
+                                .state_transition(&signed_block, validate_result, &mock_engine)
+                                .await
+                                .map_err(|err| err.to_string());
+                        }
```

Comment:
```suggestion
                        
                            result = state
                                .state_transition(&signed_block, validate_result, &mock_engine)
                                .await
                                .map_err(|err| err.to_string());
                        if result.is_err() {
                            break:
                        }
```

Wouldn't it make more sense to do this? then to continue reading blocks from disk?

Reply:
with your code change, if state_transition returns an error

```
                        let block_path = case_dir.join(format!("blocks_{}.ssz_snappy", i));
                        if !block_path.exists() {
                            panic!("Test asset not found: {:?}", block_path);
                        }
                        let signed_block: SignedBeaconBlock = utils::read_ssz_snappy(&block_path)
                            .expect(&format!("cannot find test asset (blocks_{i}.ssz_snappy)"));
```

Only these lines will be ran, which isn't doing anything for the test